### PR TITLE
Kd update se script spawn sync

### PR DESCRIPTION
**Pull request by KostiantynDrozd-okta**
In order to avoid code injections, replaced axios.get with spawnSync and curl.

### CODE REVIEW

File: scripts/update-se-drivers.js [hunk 1]
```diff
@@ -31,12 +31,17 @@ console.log(`Chrome Major Version - ${chromeMajorVersion}`);
 
 const chromeDriverUrl = `https://chromedriver.storage.googleapis.com/LATEST_RELEASE_${chromeMajorVersion}`;
 
-axios.get(chromeDriverUrl).then((response) => {
-  const chromeDriverVersion = response.data;
-  console.log(`Chrome Driver Version - ${chromeDriverVersion}`);
+const chromeDriverVersion = spawnSync('curl', [chromeDriverUrl], { encoding : 'utf8' });
```
Comment (oleksandrpravosudko-okta) [inline scripts/update-se-drivers.js:34]: nit: can destructure `stdout` and `stderr` into variables:

```javascript
const {stdout: chromeDriverVersion, stderr: errorResponse} = spawnSync('curl', [chromeDriverUrl], { encoding : 'utf8' });
```

Comment (KostiantynDrozd-okta) [reply scripts/update-se-drivers.js]: Done!

File: scripts/update-se-drivers.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const axios = require('axios');
```
Comment (oleksandrpravosudko-okta) [inline scripts/update-se-drivers.js:1]: nit can remove axios import?

**Review by oleksandrpravosudko-okta — APPROVED** [review]
It looks like `chromeDriverVersion` has already been parameterized in scope of [OKTA-367550](https://oktainc.atlassian.net/browse/OKTA-367550). It still looks valuable though to replace axios usage with an OS utility.

File: scripts/update-se-drivers.js (hunk 2 above)
Comment (KostiantynDrozd-okta) [reply scripts/update-se-drivers.js]: Done!

**Pull request closed without merging**